kpops/components/streams_bootstrap/producer/producer_app.py

```diff
@@ -106,16 +104,8 @@ async def destroy(self, dry_run: bool) -> None:
         if cluster_values:
             log.debug("Fetched Helm chart values from cluster")
             name_override = self._cleaner.helm_name_override
-            try:
-                self._cleaner.values = self.values.model_validate(cluster_values)
-                self._cleaner.values.name_override = name_override
-            except ValidationError as validation_error:
-                warning_msg = f"The values in the cluster are invalid with the current model. Falling back to the enriched values of {PIPELINE_YAML} and {DEFAULTS_YAML}"
-                log.warning(warning_msg)
-                debug_msg = f"Cluster values: {cluster_values}"
-                log.debug(debug_msg)
-                debug_msg = f"Validation error: {validation_error}"
-                log.debug(debug_msg)
+            self._cleaner.values = self.values.model_validate(cluster_values)
+            self._cleaner.values.name_override = name_override
```

Comment:
why did you revert this?

Reply:
It was effected during the rebase. Fixed it and added tests for it.